conf.d/abbr_tips.fish

```diff
@@ -31,6 +31,23 @@ function __abbr_tips --on-event fish_postexec -d "Abbreviation reminder for the
                 set -e __ABBR_TIPS_KEYS[$abb]
                 set -e __ABBR_TIPS_VALUES[$abb]
         end
+    else if test "$command[1]" = "alias"
+        # Update abbreviations list when adding aliases
+        and not contains -- "$command[2]" $__ABBR_TIPS_KEYS
+        if test (count $command) = 2
+            set command_split (string split = $command[2])
+            set -a __ABBR_TIPS_KEYS $command_split[1]
+            set -a __ABBR_TIPS_VALUES $command_split[2]
```

Comment:
I think that aliases with more than one word aren't handled because you only add the second element of `command_split`.
For example, `alias gq 'grep -q'` would only add `'grep` to the `__ABBR_TIPS_VALUES` array

Adding all elements starting from the second one is done with `set -a __ABBR_TIPS_VALUES $command_split[2..-1]` .

Also, you should take care of quotes used when creating an alias for a long command (ex : `alias gq 'grep -q'`), they should be trimmed before adding the command to `__ABBR_TIPS_VALUES`